pkg/varlog/operations.go

```diff
@@ -115,3 +115,27 @@ func (v *logImpl) logStreamMetadata(ctx context.Context, tpID types.TopicID, lsI
 	}
 	return lsd, err
 }
+
+func (v *logImpl) logStreamReplicaMetadata(ctx context.Context, tpID types.TopicID, lsID types.LogStreamID) (snpb.LogStreamReplicaMetadataDescriptor, error) {
```

Comment:
I don't know how it will be used. Don't you need an aggregation to replace LogStreamMetadata?

Reply:
We, actually don't need LogStreamMetadata at this time. Intend of the current change is to replace the RPC LogStreamMetadata with the RPC LogStreamReplicaMetadata. So, this change marks the RPC LogStreamMetadata as will be deprecated.

Then, do we need aggregation of metadata of all replicas eventually? Yes, if we need that. The current implementation of RPC LogStreamMetadata just selects the first non-error metadata rather than aggregating the metadata of all replicas. The RPC LogStreamReplicaMetadata so does that. Perhaps, we may need only the head and tail of the log stream. In this case, the aggregation of the metadata could be better.